packages/wallet/backend/src/paymentPointer/service.ts

```diff
@@ -247,4 +248,27 @@ export class PaymentPointerService implements IPaymentPointerService {
       await trx.rollback()
     }
   }
+
+  public async getExternalPaymentPointer(id: string): Promise<{
```

Comment:
We should also have route for this function so we can get the payment pointer asset on frontend as well.